pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -156,6 +158,18 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			return reconcile.Result{}, nil
 		}
 
+		// checking if secret "rook-ceph-crash-collector-keyring" is present which is required to create crashcollector pods
+		secret := &corev1.Secret{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: crashCollectorKeyName, Namespace: namespace}, secret)
+		if err != nil {
+			if kerrors.IsNotFound(err) {
+				logger.Debugf("secret %q not found. %v. retrying in 30 sec.", crashCollectorKeyName, err)
+				return reconcile.Result{Requeue: true, RequeueAfter: 30 * time.Second}, nil
```

Comment:
```suggestion
				return reconcile.Result{Requeue: true, RequeueAfter: 30 * time.Second}, nil
```

Make `30 * time.Second` a constant and use it in the logger too with `.String()`.

Reply:
done.